example/env.js

```diff
@@ -1,4 +1,4 @@
-const envalid = require('..')
+const envalid = require('../src')
```

Comment:
the old one should still work since it'll resolve `main`

Reply:
Changed them back.